tools/fix_candidate_age.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * inititalize
+ */
+require_once 'generic_includes.php';
+
+$config   = NDB_Config::singleton();
+$database = $config->getSetting('database');
+
+$instruments = Utility::getAllInstruments();
+
+$confirm=false;
+if (!empty($argv[1]) && $argv[1] == 'confirm') {
+    $confirm = true;
+}
+
+$incorrectAges = array();
+$nullAges      = array();
+
+foreach ($instruments as $inst=>$fullName) {
+
+    if (!$DB->tableExists($inst)) {
+        echo $inst." does not have a table in the Database\n";
+        continue;
+    }
+
+    //Get instrument SQL table
+    $DBInstTable = $DB->pselect(
+        "SELECT * FROM $inst",
+        array()
+    );
+
+    // Break if Date taken does not exist
+    if (!$DB->columnExists($inst, 'Date_taken')) {
+        continue;
+    }
+
+    foreach ($DBInstTable as $k => $row) {
+        // Get Instrument Instance
+        $instrument = NDB_BVL_Instrument::factory($inst, $row['CommentID'], null, false);
+        if (!$instrument) {
+            // instrument does not exist
+            continue;
+        }
+
+        $dateTaken = $row['Date_taken'];
+        $commentID = $row['CommentID'];
+
+        // Flag for problem with date
+        $trouble=false;
+        if (!empty($row['Date_taken'])) {
+            // Check if age is null OR if wrong age
+            if (empty($row['Candidate_Age'])) {
+                // Null age
+                $nullAges[$inst][$commentID] = $row;
+                $trouble=true;
+            } else {
+                // get Age from instrument class
+                $calculatedAge = $instrument->getCandidateAge();
+                $agemonths = $calculatedAge['year'] * 12 + $calculatedAge['mon'] + ($calculatedAge['day'] / 30);
```

Comment:
@ridz1208 oops... nevermind. I thought it was age in years. The Candidate_Age threw me off since the name is very misleading and it should be Candidate_Age_Months